.kitchen.yml

```diff
@@ -22,12 +22,14 @@ suites:
   - name: install
     run_list:
       - recipe[cerner_splunk_ingredient_test::splunk]
-      - recipe[cerner_splunk_ingredient_test::universal_forwarder]
   - name: uninstall
     run_list:
-      - recipe[cerner_splunk_ingredient_test::splunk]
```

Comment:
Don't we still need to install something to test the uninstall recipe works properly?